### DISCUSSION TITLE
Optimal alpha in exponential smoothing models

### DISCUSSION THREAD

**Discussion by matsuobasho** [Nixtla/statsforecast#877] [Q&A]
I realize this is likely out-of-scope for this package, but are there any ready-to-use techniques for finding the optimal alpha based on minimizing the SSE of the residuals?

I understand I can just write my own optimization code but curious what techniques practitioners turn to?

**Comment by jmoralez** [comment]
I think that's what this model does https://github.com/Nixtla/statsforecast/blob/7f60571b1242413b372028504e60b56d0d566214/statsforecast/models.py#L1989
do you want to extract the value of alpha?

**Comment by matsuobasho** [comment]
Ah, great, I should have looked at those functions before asking.  Extracting alpha would be beneficial - is that feature available in that function?

**Reply by jmoralez** [reply]
No, the alpha is found here https://github.com/Nixtla/statsforecast/blob/7f60571b1242413b372028504e60b56d0d566214/statsforecast/models.py#L1783-L1791 but it isn't stored. You can open a feature request for us to store it in that model if that'd be useful for you.

**Comment by matsuobasho** [comment]
Thanks @jmoralez will do if I actually have need for it.  This answers my initial request, glad these functions are available!
